src/IonTests.ts

```diff
@@ -20,4 +20,6 @@ export { Import } from './IonImport';
 export { LocalSymbolTable } from './IonLocalSymbolTable';
 export { LowLevelBinaryWriter } from './IonLowLevelBinaryWriter';
 export { NullNode } from './IonBinaryWriter';
-export { BinaryWriter } from "./IonBinaryWriter";
\ No newline at end of file
+export { BinaryWriter } from "./IonBinaryWriter";
+export { ParserBinaryRaw } from "./IonParserBinaryRaw";
```

Comment:
This makes the `ParserBinaryRaw` and `BinarySpan` classes accessible from within the unit test framework.